include/unk_0206015C.h

```diff
@@ -0,0 +1,60 @@
+#include "global.h"
```

Comment:
it's probably somewhat better to only put the includes required by the .h in the .h, and the rest of the includes in the .c